tests/limma/test_lmFit.py

```diff
@@ -59,3 +59,71 @@ def test_lmFit(self):
             ]
         )
         self.assertTrue(np.allclose(fit.coefficients, coef_ref, rtol=1e-6))
+
+    def test_contrasts_fit(self):
```

Comment:
Why is this test in test_lmFit and not in test_constrasts? I am just wondering because the function is defined in contrasts.py

Reply:
The input data for `contrasts_fit` are similar to those of `lmFit`, so it makes sense to share the input data (generated in `setUp` method).
Maybe the file should be renamed (_e.g._ to `test_fit.py`)